week-03/README.md

```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
 
  
 [Course Textbook Chapter 3, Problem 4]
-
+- *Question* (sbagdi-stat6250): What are the different types of syntax errors?
 
 
 [Course Textbook Chapter 3, Problem 5]
-
+- *Question* (sbagdi-stat6250): How to locate a statement that lacks a semi-colon in an error message?
 
 
 [Course Textbook Chapter 3, Problem 6]
-
+- *Question* (sbagdi-stat6250): Why doesn’t simply adding a quotation mark (in missing quotation mark errors) and resubmitting a program resolves the problem? 
 
 
 [Course Textbook Chapter 3, Problem 7]
-
+- *Question* (sbagdi-stat6250): What are the essential steps to resolve the issue of invalid option?
 
 
 [Course Textbook Chapter 3, Problem 10]
+- *Question* (sbagdi-stat6250): Which syntax error, other than the ‘missing run statement’, results in error message- ‘DATA Step running’ at the top of active window?
 
 
 
-[Course Textbook Chapter 4, Problem 1]
 
+[Course Textbook Chapter 4, Problem 1]
+- *Question* (sbagdi-stat6250): Which option is used to remove the default observation column (column that displays observation numbers)? Where do the programmer specifies that option in his program?
 
 
 [Course Textbook Chapter 4, Problem 3]
+- *Question* (sbagdi-stat6250): Why are logical operators used in any SAS programs? Also, Which operator can be used as an alternative to ‘OR’ operator? 
+- *Answer* (sbagdi-stat6250): The logical operators link sequence of expressions into compound expressions. Two types are: ‘AND’ (&) and ‘OR’ (|). Operator ‘IN’ can be used in place of ‘OR’ operator. 
 
 
 
 [Course Textbook Chapter 4, Problem 4]
-
+- *Question* (sbagdi-stat6250): What happens if a programmer doesn’t use OUT= option in a PROC SORT step? 
 
 
 [Course Textbook Chapter 4, Problem 7]
+- *Question* (sbagdi-stat6250): What does the DESCENDING option does? And to which statement should it be added?
+a) The DESCENDING option sorts observations in descending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the BY statement.
+b) The DESCENDING option sorts observations in ascending order of second variable (the one programmer specifies in VAR statement) within descending order of the first variable. And it should be added to the BY statement.
+c) The DESCENDING option sorts observations in ascending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the PROC PRINT statement.
+d) The DESCENDING option sorts observations in ascending order of first variable (the one programmer specifies in VAR statement) within descending order of the second variable. And it should be added to the PROC SORT statement.
 
 
 
 [Course Textbook Chapter 4, Problem 9]
+- *Question* (sbagdi-stat6250): Which operator is used to select observations that include a specified string?
```

Comment:
Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.